packages/ember-debug/lib/warn.js

```diff
@@ -51,10 +50,13 @@ if (DEBUG) {
   };
 
   registerHandler(function logWarning(message) {
-    Logger.warn(`WARNING: ${message}`);
-    if ('trace' in Logger) {
-      Logger.trace();
-    }
+    /* eslint-disable no-console */
+    console.warn(`WARNING: ${message}`); //eslint-disable-line no-console
```

Comment:
Seems like this comment isn’t needed (since the surrounding ones should do the trick)